.goreleaser.docker.yaml

```diff
@@ -127,7 +124,7 @@ dockers:
     goos: 'linux'
     goarch: 'amd64'
     image_templates:
-      - '{{ .Env.REGISTRY }}/jvs-cert-rotation:{{ .Version }}-amd64'
```

Comment:
This is intentional. goreleaser has its own way to detect `{{ .Version }}` and we should keep that behavior. Why change?

Reply:
To confirm, .Version will pick up github.sha automatically? since DOCKER_TAG = github.sha, so I am trying to use DOCKER_TAG everywhere.

Reply:
Right. Let's goreleaser do its job and keep `{{ .Version }}`. `DOCKER_TAG` can still override it.

Reply:
Tried use .Version, it seems the order is .Version = git tag, if tag is absent then commit sha. see integration run https://github.com/abcxyz/jvs/actions/runs/4625818241/jobs/8181935474. We won't be about to override it with DOCKER_TAG, we can only override it with a git tag. I think it makes more sense to just use DOCKER_TAG to make it obvious. What do you think?

Reply:
That was intended behavior. I'm not sure what problem you're trying to solve with DOCKER_TAG

Reply:
Tried a few more scenarios, so goreleaser requires a tag, therefore the .Version will always be the tag, and the tag needs to comply with the version semantics, and git sha is not a valid tag here. Use DOCKER_TAG (github.sha) to tag the image, we can easily copy what is already built in ci when releasing. similar to [github metrics](https://github.com/abcxyz/github-metrics-aggregator/blob/main/.github/workflows/release.yml#L24) there are probably other ways too, I would prefer to stick with the existing pattern. 

Slightly deviating from this topic, we can improve the ci by reducing the ci build time by only build the amd images, currently it takes too long for all the images build complete.

Reply:
Forgot to check Lumberjack example! Yes, that works too, updating, thx!